### DISCUSSION TITLE
DLL export interface for Metrics

### DISCUSSION THREAD

**Discussion by TomSvk88** [open-telemetry/opentelemetry-cpp#2311] [Q&A] [answered]
Hi,

I can see from the code itself that Traces and Logs classes as exported as dll interface but Metrics classes are not. Is there any particular reason for that or is it just it was not added yet?
Thank you for any response.

Tomas

**Comment by lalitb** [comment, marked as answer]
It was just not added yet. In general, contributions are welcome to add the support.
